docs/transport/braginskii.rst

```diff
@@ -1,4 +1,5 @@
 .. _braginskii
+.. py:module:: physics.transport.braginskii
```

Comment:
This puts the module in the [Global Module Index (live version of docs)](http://docs.plasmapy.org/en/master/py-modindex.html), which, incidentally, is a mess right now.